tests/_craft/migrations/m220617_180419_add_fields.php

```diff
@@ -545,6 +548,149 @@ public function createCheckboxesField(int $groupId): Checkboxes
         return $checkboxes;
     }
 
+    public function createLinkitField(int $groupId): LinkitField
+    {
+        $linkitField = new LinkitField([
+            'groupId' => $groupId,
+            'name' => 'linkit',
+            'handle' => 'linkit',
+            'instructions' => 'Default Instructions
+Helper text to guide the author.',
```

Comment:
is it possible for a string to span multiple lines like this? same on line 653 (i'm not familiar enough with PHP to know)

Reply:
extra break removed